core/libs/models/src/tree.rs

```diff
@@ -0,0 +1,298 @@
+use crate::file_metadata::FileType;
```

Comment:
none of these fn's had unit tests?

Reply:
https://github.com/lockbook/lockbook/blob/master/core/src/pure_functions/files.rs#L448

they just test `apply`